classify.go

```diff
@@ -29,15 +29,24 @@ type Confidence struct {
 	// The associated confidence with the label.
 	Confidence float32 `json:"confidence"`
 }
+
+type LabelConfidence struct {
```

Comment:
why is `labels = {SOME_LABEL: {confidence: SOME_VAL}}` rather than `labels = {SOME_LABEL: SOME_VAL}`? I'm assuming that we might add extra properties along with the confidence in the future? 
If that's the case, I think the `LabelConfidence` name could be better named as `LabelProperties` or something similar? (this is not important of course since it won't affect functionality, just easier intellisense)

Reply:
I agree, i think in retrospect this should have been implemented like that on the backend. But since we can't change the backend without introducing a breaking change, we just have to mirror it in the SDK. Good call on renaming LabelConfidence to LabelProperties, i'll do that